### DISCUSSION TITLE
How to use go_router with this?

### DISCUSSION THREAD

**Discussion by mika76** [bdlukaa/fluent_ui#222] [Q&A] [answered]
I'm pretty new to flutter and am trying to do my first little test app using Windows fluent ui. 

My basic setup currently is a`NavigationView > NavigationBody > ?`

So in the docs go_router seems to use a `MaterialApp.router` widget to actually display the current route.

How do I do this without using MaterialApp.router?

(I realise this is a question mostly about go_router but it's very specific to fluid_ui so I thought it would be better to ask here than on the go_router repo)

**Comment by mika76** [comment, marked as answer]
Right so to update here because someone might learn from this. I had mistakenly thought that go_router works like a router on the web when you have a "view" which would show a widget. Well it seems that go_router actually changes the main app only so you sould have to redo all the navigation and scaffolding for each route. 

Also there is a `FluentApp.router` to use this with fluent_ui if this is the "route" you wanna go 😉 

(Also I'm still learning all this so if this is wrong please let me know)

**Reply by BTMuli** [reply]
> Well it seems that go_router actually changes the main app only so you sould have to redo all the navigation and scaffolding for each route.

Is there any code for refer?
